rixs/process2d.py

```diff
@@ -217,6 +219,7 @@ def photon_events_to_image(photon_events, bins=None):
         1D vector describing column position
     y_centers : array
         1D vector describing row position
+        +ve y is up convetion is applied.
```

Comment:
convention is mispelt here

Reply:
This is corrected in the subsequent commit.